CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
+## 1.3.0 (April 8, 2020)
```

Comment:
as now we have one release notes per platform and components maybe it makes sense to date changes same as platform monthly release (May 7)? or at least we need to update date as sprint last day (April 23)